src/Microsoft.AspNet.Mvc.Core/Resources.resx

```diff
@@ -460,4 +457,7 @@
   <data name="TempData_CannotSerializeDictionary" xml:space="preserve">
     <value>The '{0}' cannot serialize a dictionary with a key of type '{1}' to session state.</value>
   </data>
+  <data name="ValueInterfaceAbstractOrOpenGenericTypesCannotBeActivated" xml:space="preserve">
+    <value>Value, Interface, Abstract or Open Generic types cannot be activated by '{0}'.</value>
```

Comment:
How about: `The type '{0}' cannot be created by '{1}' because it is either a value type, an interface, an abstract class, or an open generic type.`